### DISCUSSION TITLE
Autogen with AWS Lambda

### DISCUSSION THREAD

**Discussion by mseefelder2** [microsoft/autogen#1099] [Q&A] [answered]
Hi, I'm trying to write a Lambda function using Autogen, but I'm not being able to create a Layer of Autogen that works on the Lambda Linux environment.

I've even pip installed pyautogen on a Linux environment and then created a Layer from it, but I'm getting the following error:

"Runtime.ImportModuleError: Unable to import module 'lambda_function': Error importing numpy: you should not try to import numpy fromits source directory; please exit the numpy source tree, and relaunch your python interpreter from there."

Any ideas?

**Comment by sonichi** [comment]
What's the python version? I know py3.12 is not supported.

**Reply by mseefelder2** [reply]
3.10

**Comment by ekzhu** [comment]
Doesn't look like an issue related to AutoGen. Have you tried running the autogen part of the code locally?

**Comment by mseefelder2** [comment, marked as answer]
Hi, I've just discovered what the problem was. I was assuming that my Python version was 3.10, but the EC2 Linux I was running was 3.9. Now it is working at last!

**Comment by amanuel1782** [comment]
Hi, I'm trying to write a Lambda function using Autogen, but I have "issue with Cache" unable to solve the cache issue any suggestion

**Reply by sonichi** [reply]
You can set {"cache_seed": None} in the `llm_config`.

**Reply by ekzhu** [reply]
@sonichi let's consider removing cache by default #2133
